ecommerce/pricing/lib/pricing/commands.rb

```diff
@@ -45,6 +52,16 @@ class RegisterCoupon < Infra::Command
     alias aggregate_id coupon_id
   end
 
+  class CreateHappyHour < Infra::Command
+    attribute :id, Infra::Types::UUID.meta(omittable: true)
+    attribute :product_ids, Infra::Types::Array.of(Infra::Types::UUID)
+    attribute :name, Infra::Types::String
+    attribute :code, Infra::Types::String
```

Comment:
what is code used for?

Reply:
It's probably a bias from my other projects. 

From my experience `code` - a short and simplified version of the `name` is often useful for internal usage. The `id` (especially when it's `uuid`) is hard to remember and use by humans. The `name` is more like a label, often includes whitespaces or other special chars and is unwieldy (and can change). The code in this case is kind of a slug that can be used for quickly identifying some object.

However, if you prefer to remove it, I'm totally fine with it.